### PR TITLE
Circle ci

### PR DESCRIPTION
**Pull request by flovilmart**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,47 +7,13 @@ os: osx
 osx_image: xcode8.3
 cache:
   - cocoapods
-
+florentvilmart@me.com
```
Comment (montymxb) [inline .travis.yml]: This is causing a syntax for your travis.yml. There are a couple other things I noticed in lint that I still need to look at here.

**Comment by flovilmart** [comment]
This one will go away actually, superseded by the next PR

**Comment by flovilmart** [comment]
@montymxb this should be good now :)

**Review by montymxb — APPROVED** [review]
Awesome :octocat: !

**Pull request merged**